src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/EnvelopeProcessor.java

```diff
@@ -147,6 +150,28 @@ public static void assertEnvelopeHasPdfs(InputEnvelope envelope, List<Pdf> pdfs)
         }
     }
 
+    /**
+     * Assert envelope contains at least one scannable item
+     * with valid document type and ocr data for new application.
+     * Throws exception otherwise.
+     *
+     * @param envelope to assert against
+     */
+    public static void assertEnvelopeContainsOcrDataForNewApplication(InputEnvelope envelope) {
+
+        if (Classification.NEW_APPLICATION.equals(envelope.classification)) {
+
+            boolean isValidScannableItems = envelope.scannableItems
+                .stream()
+                .anyMatch(item -> item.documentType.equals(InputDocumentType.SSCS1) && item.ocrData.isEmpty());
```

Comment:
so `isValidScannableItems` means specific doc type and empty ocr data. isn't the requirement for ocr data to be present instead? if yes then giving such conditions as is now, the variable proposes to be `isNotValid...`

Reply:
No, ocr data shouldn't be empty. It is also missing jurisdiction condition. I'll push the changes soon